frontend/src/utils/index.js

```diff
@@ -628,6 +628,22 @@ export function allErrorCodesFromLastErrors (lastErrors) {
   return uniq(compact(flatMap(lastErrors, 'codes')))
 }
 
+export const allMemberRoles = [
```

Comment:
MEMBER_ROLE_DESCRIPTORS or something like this. Do we need this in utils?

Reply:
renamed to MEMBER_ROLE_DESCRIPTORS, keep it in utils for now will move to store when we read custom roles